src/Components/NavigationBar/NavigationBar.tsx

```diff
@@ -1,18 +1,64 @@
+                    <a class='pr-10' href='https://deriv.com/about'>About us</a>
+                    <a class='pr-10' href="https://deriv.com/contact-us">Contact us</a>
+                </div>
+            </div>
+            <header class='h-[73px] w-4/5 mobile:h-[48px] mobile:pl-4 mobile:w-full mobile:flex'>
+                <section class="flex items-center">
+                    <img class="desktop:hidden mobile:mr-2.5" src={`/src/assets/${showMobileNav() ? "close.svg" : "hamburger_menu.svg"}`} alt="Deriv Logo" width="16" height="16" onclick={() => setShowMobileNav(!showMobileNav())} />
+                    <NavLink class="flex items-center text-white" href="/">
+                        <img src='/src/assets/deriv.svg' class="h-[73px] w-[180px] mobile:h-auto mobile:w-[84px]" height='73' width="180" alt="Deriv Logo" />
+                        <h1 class='inline ml-6 text-2xl  mobile:text-base mobile:ml-2'>API</h1>
+                    </NavLink>
+                    <nav class="flex flex-row ml-5% mobile:hidden">
+                        {navs().map(n => <NavLink activeClass="after:active-page after:navigation-transition" href={n.path} class="relative text-base mr-10 text-white before:active-page hover:before:navigation-transition" end={n.path === "/"}>{n.label}</NavLink>)}
+                    </nav>
+                </section>
+            </header>
+            <section class={`fixed bg-white top-[48px] h-[calc(100%+54px) w-[253px] overflow-hidden left-[-254px] text-base	burger ${showMobileNav() && "transition-left"}`}>
+                <div class="flex flex-col h-full py-6 pr-4 pl-5">
+                    <div class="menu-wrapper">
+                        {navs().map(n => {
+                            if (n.path === "/docs")
+                                return (<div class="py-2.5 px-2">
+                                    <div class="font-bold px-4 items-center	" onclick={() => {
+                                        setShowDocumentationItems(!showDocumentationItems())
+                                    }}>
+                                        {n.label}
+                                        <img width="16" height="16" class="float-right" src="/src/assets/arrow_down.svg" />
+                                    </div>
+                                    <Show when={showDocumentationItems()}><div class="px-4">{docs_nav}</div></Show>
+                                </div>
+                                )
+                            else
+                                return (
+                                    <div class="py-2.5 px-2 align-middle">
+                                        <NavLink activeClass="before:mobile-nav-after " class="font-bold flex relative items-center px-4" href={n.path} end>{n.label}</NavLink>
+                                    </div>
+                                )
+                        }
+                        )}
```

Comment:
Some q here with <Switch /> <Match /> instead of if else: https://www.solidjs.com/docs/latest/api#%3Cswitch%3E%2F%3Cmatch%3E

Reply:
I changed that using show condition. They are identical IMO